docker-compose.yml

```diff
@@ -6,38 +6,56 @@ services:
         command: >
             /bin/bash -c "
                 until echo > /dev/tcp/postgres/5432; do sleep 1; done
-                npx pubsweet server
+                node app
             "
         ports:
             - 3000:3000
         depends_on:
             - postgres
+        env_file:
```

Comment:
Is this intended to pass the variables in `.env` to the `app` container? This file contains all sort of variables interpolated into the `docker-compose*.yml` files, but they are not normally passed directly to the containers.

They can be passed through in this way, I think:
https://docs.docker.com/compose/environment-variables/#pass-environment-variables-to-containers
or in general by making them explicit in `environment`. But this service should not need `SCIENCEBEAM_TAG`, `GROBID_TAG`, `COMPOSE_PROJECT_NAME`, etc. which should remain only available to this particular file.

Reply:
Yes, I was just being lazy. I'll make it explicit instead. 👍